api/opencl_platform_layer.asciidoc

```diff
@@ -1606,6 +1606,9 @@ include::{generated}/api/version-notes/CL_DEVICE_ATOMIC_MEMORY_CAPABILITIES.asci
         {CL_DEVICE_ATOMIC_ORDER_RELAXED} \| +
         {CL_DEVICE_ATOMIC_SCOPE_WORK_GROUP}
 
+        A device that does not support CL_DEVICE_SVM_ATOMICS (and hence does not support CL_MEM_SVM_ATOMICS) may still support CL_DEVICE_ATOMIC_SCOPE_ALL_DEVICES. On these devices, an atomic operation with
+        memory_scope_all_svm_devices will behave the same as if the scope were memory_scope_device - refer to <<memory-consistency-model, memory consistency model>>
```

Comment:
Thanks!  Latest text LGTM.  Here are my suggested formatting changes.  If these look good, just accept the suggestion, and I'll merge.

```suggestion
        A device that does not support {CL_DEVICE_SVM_ATOMICS} (and hence does not support {CL_MEM_SVM_ATOMICS}) may still support {CL_DEVICE_ATOMIC_SCOPE_ALL_DEVICES}. On these devices, an atomic operation with
        *memory_scope_all_svm_devices* will behave the same as if the scope were *memory_scope_device* - refer to the <<memory-consistency-model, memory consistency model>>.
```